### DISCUSSION TITLE
auto scroll to element using outline menu

### DISCUSSION THREAD

**Discussion by m44kubica** [prabhuignoto/react-chrono#299] [Q&A]
Hi,

is it possible to scroll the page to the element selected in outline menu?
I've gone through documentation and haven't found anything helpful.

If I've missed anything, I'm sorry.

Kind regards,
Marcin

**Comment by prabhuignoto** [comment]
@m44kubica yes, clicking on an outline menu would auto scroll the timeline and bring the selected item in focus. Is this not working for you? please make sure the `enableOutline` prop is set.


https://user-images.githubusercontent.com/2144065/200933140-d43db32f-8969-41b1-8fe4-7f9976b15c63.mp4
